File: qual.py
t = int(input())
l = 0
c = []
for i in range(0, t):
	n, k = int(input().split)
	k = int(input())
	a = []
	for j in range(0, n):
		x = int(input())
		a.append(x)
	a.sort(reverse=True)
	count = 0
	d = a[k-1]
	for j in range(0, n):
		if a[j] == d or a[j] > d :
			count=count+1
	c.append(count)
	l = l+1

for i in range(0, t):
	print(c[i])

File: qual.cpp
	#include<bits/stdc++.h>
	using namespace std;
	int main()
	{
	int t;
	cin>>t;
	int c[t], l=0, test=t;
		while(t--)
		{
			int n, k;	
			cin>>n>>k;
			double a[n];
			for(int i=0;i<n;i++)
				cin>>a[i];

			int count=0;
		sort(a, a+n, greater<int>());

					int d=a[k-1];
					for(int i=0;i<n;i++)
						if(a[i]==d || a[i]>d)
						count++;
		c[l++]=count;
		}
		for (int i = 0; i < test; ++i)
		{
			cout<<c[i]<<endl;
		}
	return 0;
	}